source/brailleDisplayDrivers/handyTech.py

```diff
@@ -581,10 +602,47 @@ def __init__(self, port="auto"):
 			self._dev.close()
 
 		else:
+			wx.CallAfter(self.destroy_message_window)
 			raise RuntimeError("No Handy Tech display found")
 
+	def create_message_window(self):
+		try:
+			self._messageWindow = InvisibleDriverWindow(self)
+		except:
+			log.debugWarning("", exc_info=True)
+	def destroy_message_window(self):
+		try:

```

Comment:
Don't forget to reset the counter here. In some situations when the driver is re-initialised, the same instance as before is reused. It could be left in a bad state if re-init takes place when the counter is > 0.